apps/prairielearn/src/pages/studentAssessment/studentAssessment.ts

```diff
@@ -24,36 +22,10 @@ import { checkPasswordOrRedirect } from '../../middlewares/studentAssessmentAcce
 import { StudentAssessment } from './studentAssessment.html.js';
 
 const router = Router();
-const sql = loadSqlEquiv(import.meta.url);
```

Comment:
This SQL file is no longer referenced; you should delete it if it's unused.

Reply:
Deleted the unused SQL file in [43e98f0](https://github.com/PrairieLearn/PrairieLearn/pull/10889/commits/43e98f06be2e3d1eaa245e359fef82b33e896ca2).